book_source/03_topical_pages/03_pecan_xml.Rmd

```diff
@@ -752,9 +752,77 @@ The following tags can be used for state data assimilation. More detailed inform
 	<process.variance>FALSE</process.variance>
   <sample.parameters>FALSE</sample.parameters>
   <state.variables>
-   <variable>AGB.pft</variable>
-   <variable>TotSoilCarb</variable>
+   <variable>
+    <variable.name>AbvGrndWood</variable.name>
+    <unit>MgC/ha</unit>
+    <min_value>0</min_value>
+    <max_value>9999</max_value>
+   </variable>
+   <variable>
+    <variable.name>LAI</variable.name>
+    <unit></unit>
+    <min_value>0</min_value>
+    <max_value>9999</max_value>
+   </variable>
+   <variable>
+  <variable.name>SoilMoistFrac</variable.name>
+  <unit></unit>
+  <min_value>0</min_value>
+  <max_value>100</max_value>
+  </variable>
+   <variable>
+    <variable.name>TotSoilCarb</variable.name>
+    <unit>kg/m^2</unit>
+    <min_value>0</min_value>
+    <max_value>9999</max_value>
+   </variable>
   </state.variables>
+  <Obs_Prep>
+   <AGB>
+    <Var>AbvGrndWood</Var>
+    <AGB_dir>/projectnb/dietzelab/dongchen/Multi-site/download_500_sites/AGB</AGB_dir>
+    <Time_Step>
+     <unit>year</unit>
+     <num>1</num>
+    </Time_Step>
+    <Allow_download>TRUE</Allow_download>
+    <Out_dir>/projectnb/dietzelab/dongchen/All_NEON_SDA/test_OBS</Out_dir>
+    <Export_CSV>TRUE</Export_CSV>
+    <buffer></buffer>
+   </AGB>
+   <LAI>
+    <Var>LAI</Var>
+    <Out_dir>/projectnb/dietzelab/dongchen/All_NEON_SDA/test_OBS</Out_dir>
+    <Search_Window>30</Search_Window>
+    <NCore></NCore>
+    <Time_Step>
+     <unit>year</unit>
+     <num>1</num>
+    </Time_Step>
+    <Export_CSV>TRUE</Export_CSV>
+   </LAI>
+   <SMP>
+    <Var>SMP</Var>
+    <Out_dir>/projectnb/dietzelab/dongchen/All_NEON_SDA/test_OBS</Out_dir>
+    <Search_Window>30</Search_Window>
+    <Time_Step>
+     <unit>year</unit>
+     <num>1</num>
+    </Time_Step>
+    <Export_CSV>TRUE</Export_CSV>
+    <Update_CSV>FALSE</Update_CSV>
+   </SMP>
+   <SoilC>
+    <Var>SoilC</Var>
+    <Out_dir>/projectnb/dietzelab/dongchen/All_NEON_SDA/test_OBS</Out_dir>
+    <Time_Step>
+     <unit>year</unit>
+     <num>1</num>
+    </Time_Step>
+   </SoilC>
+   <Start_Date>2012-07-15</Start_Date>
```

Comment:
1. should be `start.date` for consistency
2. the meaning of this arg is not currently documented
3. how is this the same/different from the overall `<state.data.assimilation><start.date>`? This variable seems redundant.